controllers/users.js

```diff
@@ -431,6 +427,8 @@ class Users {
         }
       );
 
+      subscribe(id, userExists.id);
```

Comment:
When a user is followed you add the follower as a subscriber but when he/she is unfollowed you don't remove that follower in the subscribers. After unfollowing the author, I was still receiving emails!